masq/src/commands/financials_command.rs

```diff
@@ -1696,11 +1758,31 @@ mod tests {
     }
 
     #[test]
-    #[should_panic(
-        expected = "Broken code: Clap validation should have caught this overflow of 40000000000000000000 earlier"
-    )]
-    fn parse_integer_overflow_indicates_broken_code() {
-        let _: Result<u64, String> = FinancialsCommand::parse_integer("40000000000000000000");
+    fn parse_integer_overflow_indicates_always_too_big_number_supplied() {
```

Comment:
This would be a more powerful test if it used a value just one outside the limits.